README.MD

```diff
@@ -0,0 +1,34 @@
+
+# Background
+The vTPM solution provides trusted computing capabilities for tenants. Based on trusted server hardware, the trust chain is extended from the host to the guest, creating a secure system based on hardware and virtual trust roots.
+
+Confidential computing is a new computing model that protects and isolates data in use by executing the computation process in a hardware-based Trusted Execution Environment (TEE). It prevents privileged cloud infrastructure providers from unauthorized access to tenant applications and data.
+
+However, there are still some issues. First, the TEE guest, as a complete system, still needs the ability to achieve trusted boot through measured boot and extend the trust chain to the entire TEE guest. Second, although the trust chain of the TEE guest can indeed be established without relying on the TPM hardware trust root, not all TEE hardware trust roots currently support dynamic measurement capabilities (such as AMD SEV series). The lack of this capability makes it impossible to extend and propagate the trust chain within the TEE guest system, and it is also impossible to measure and verify dynamically loaded components, greatly reducing the trustworthiness of the TEE guest.
+
+Therefore, to meet the requirements of tenants for secure storage of sensitive data, the vTPM architecture must also propose new solutions in the confidential computing scenario.
+
+# Architectural
+Confidential vtpm currently consists of three components: TPM front-end, TPM back-end, and Verifier, which is responsible for verifying the validity of EK/AK.
+![architecture](./docs/architecture.svg)
```

Comment:
Please delete ` which is responsible for verifying the validity of EK/AK.` since we have explain it in the `## Verifier` section